cmd/runner/main.go

```diff
@@ -15,26 +15,41 @@ package main
 
 import (
 	"flag"
+	"fmt"
+	"io/ioutil"
 	"log"
 	"time"
 
 	"github.com/grpc/test-infra/tools/runner"
 )
 
+// defaultOutputSuiteName provides a default name for the testsuites tag
+// in an XML report. It is based on the number of nanoseconds since the
+// UNIX epoch.
+var defaultOutputSuiteName = fmt.Sprintf("benchmarks-%d", time.Now().UnixNano())
+
 func main() {
 	var i runner.FileNames
+	var o runner.FileNames
 	var c runner.ConcurrencyLevels
 	var a string
 	var p time.Duration
 	var retries uint
+	var outputSuiteName string
 
 	flag.Var(&i, "i", "input files containing load test configurations")
 	flag.Var(&c, "c", "concurrency level, in the form [<queue name>:]<concurrency level>")
+	flag.Var(&o, "o", "name of the output file for junit xml report")
 	flag.StringVar(&a, "annotation-key", "pool", "annotation key to parse for queue assignment")
+	flag.StringVar(&outputSuiteName, "output-suite-name", defaultOutputSuiteName, "name field for testsuites in junit xml report")
```

Comment:
I wonder how useful it is to be able to specify this name. If it can be empty, I would make the default an empty string. If it cannot, I would suggest making this flag required.

If it is useful to specify this name, I would rename the flag to something like "suites-name" (so it is clear it is not the name of a single suite, and it is not something you output, it is something you set). I would change the description to "suites name for junit xml report".

I would also look into the ordering of variable and flag declarations, so they reasonably match each other (i, o, c versiosn i, c, o right now).

Reply:
Generally it is a best practice to have a id for each XML field. The schema specified here: https://github.com/windyroad/JUnit-Schema/blob/162a883ac631f61ca1d14a3d472e1fbb53c1c05f/JUnit.xsd#L33-L37 does not appear to require one for `<testsuites>` only `<testsuite>`. I do think it will directly correlate with what we output when we modify to produce multiple files like @jtattermusch requested. When this happens, I don't think this flag will be required. I'll leave it for now though.